cli/cmd/plugin/diagnostics/README.md

```diff
@@ -71,9 +71,9 @@ Flags:
       --management-cluster-name string         The name of the management cluster (required)
       --management-cluster-skip                If true, skips management cluster diagnostics
       --output-dir string                      Output directory for collected bundle (default "./")
-      --standalone-cluster-context string      The context name of the standalone cluster
-      --standalone-cluster-kubeconfig string   The standalone cluster config file (required) (default "${HOME}/.kube/config")
-      --standalone-cluster-name string         The name for the standalone cluster (required)
+      --unmanaged-cluster-context string      The context name of the standalone cluster
```

Comment:
Should we update the flag descriptions too?

Reply:
Also looks like there needs to be one more space before the description so it aligns with the rest of them.

Reply:
Whomp whomp, `%s/` has failed me!!! Thanks for the thoughtful review

Reply:
Updated - built the binary and replaced the whole section with what was generated by `--help`. So should accurately reflect spacing and what the command helps are